Eat_and_Greet/src/screens/EventScreens/post.tsx

```diff
@@ -0,0 +1,112 @@
+import { Text, View } from 'react-native';
+import { StatusBar } from 'expo-status-bar';
+
+import { Toolbar } from '../../comps/Toolbar/toolbar';
+
+import styles from '../../style';
+import React from 'react'
+import RoundedHomeButton from '../../comps/RoundedHomeButton/RoundedHomeButton';
+import { StyleSheet } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+
+export default function PostEvent() {
+  return (
+    <SafeAreaView style={styles.ScreenContainer}>
+      <SafeAreaView style={postStyle.InfoContainer}>
+        <View style={postStyle.TitleCategoryContainer}>
+          <Text style = {postStyle.LabelText}> Title {'\n'}</Text>
+          <View style={postStyle.box}>
+            <Text> Enter Title Here...</Text>
+          </View>
+        </View>
+        <View style={postStyle.TitleCategoryContainer}>
+          <Text style = {postStyle.LabelText}> Category {'\n'}</Text>
+          <View style={postStyle.box}>
+            <Text> Enter Category Here...</Text>
+          </View>
+        </View><View style={postStyle.ImageContainer}>

```

Comment:
Can probably replace this with the ChooseImageButton comp, but doesn't need to be now.

Reply:
ChooseImageButton is a circle, this seems to be a rectangle. I will be making a component later for an image to fill in a rectangle since we use that functionality a lot.